index.js

```
@@ -0,0 +1,97 @@
+        'HEAD',
+      ],
+      directory: data.targetRootDir,
+    });
+
+    // diffList B (source HEAD to Sha)
+    const sourceDiffList = await gitUtil.diff({
+      options: [
+        '--name-status',
+        data.sourceRevisionAtLastExport,
+        'HEAD',
+      ],
+      directory: data.sourceRootDir,
+    });
+
+    // diffList C (source Sha to Head)
+    const targetAndSourceDiff = await gitUtil.diff({
+      options: [
+        '--no-index',
+        '--name-status',
+        data.targetSubDirectory,
+        data.sourceSubDirectory,
+      ],
+      directory: data.tempDirPath,
+    });
+
+    const fileStatusManager = new FileStatusManager({
+      tempDirPath: data.tempDirPath,
+      targetRootDir: data.targetRootDir,
+      sourceRootDir: data.sourceRootDir,
+      sourceDirectory: data.sourceDirectory,
+      targetDirectory: data.targetDirectory,
+      sourceExcludes: data.sourceExcludes,
+      targetDiffList,
+      sourceDiffList,
+      targetAndSourceDiff
+    });
+
+    const fileOutcomes = await fileStatusManager.init();
+
+    const fileExporter = new FileExporter({
+      curationLogsPath: data.curationLogsPath,
+      sourceDirectory: data.sourceDirectory,
+      targetDirectory: data.targetDirectory,
+      exportDateTime: data.timestampForExport, // TODO format
+      fileOutcomes
+    });
+
+    await fileExporter.init();
+
+    await gitUtil.commitAndPushRemoteBranch();
+
+
+    if (argv.pullRequest) {
+      const prManager = createPrManager({
+        ghConfig: githubConfig,
+        implConfig: implementationConfig
+      });
+
+      await prManager.pushPullRequest({
+        branchName: gitUtil.targetBranch,
+        sourceSha: '', // TODO pass in data.sourceRevisionAtLastExport
+        targetSha: '', // TODO pass in data.targetRevisionAtLastExport
+        implementatorName: implementationConfig.implementatorName,
+        outcomes: fileOutcomes
+      });
+    }
+
+  })();
+} catch (error) {
```

We can do this in a follow up pr but I think we remove this try-catch since the async function will swallow any errors that get thrown in its body.